src/Geta.Optimizely.ProductFeed.Google/GoogleFeedDescriptor.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Geta Digital. All rights reserved.
+// Licensed under Apache-2.0. See the LICENSE file in the project root for more information
+
+using Geta.Optimizely.ProductFeed.Configuration;
+
+namespace Geta.Optimizely.ProductFeed.Google
+{
+    public class GoogleFeedDescriptor : FeedDescriptor
+    {
+        public GoogleFeedDescriptor() : base("google", "/googleproductfeed", "application/xml")
```

Comment:
Do this with configuration instead of hardcoded? (can be part of multisite/multi-catalog task)
Also bit weird to have `/googleproductfeed` as filename

Reply:
it's done via setup:

```csharp
services
    .AddProductFeed()
    .AddGoogleProductFeed(descriptor =>
    {
        descriptor.FileName = "/my-own-folder/my-feed";
        descriptor.SetMapper<FeedEntityMapper>();
    });
```